.github/workflows/codeql.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [ "master", "develop" ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ "master", "develop" ]
+    branches: [ "master" ]
```

Comment:
why not develop here? most PRs should be targeting develop.
not running this step for such PRs will cause headaches during release cycles.

Reply:
The problem with develop on pull requests means that every time a new commit is submitted to an already opened pull request codeQL will be run again. 

It will already be run on every PR targeting main and on develop and master as well, and on a loop. 

Simply put, this doesn't remove any coverage but helps prevent an overwhelming load of repeated loops. 

However, if it is deemed that this should be needed, I'm fine reverting this change.